### ISSUE TITLE
YouTube loudness equalization feature has stopped working

### ISSUE THREAD

**Issue by RPGWiZ4RD** [ParticleCore/Iridium#705]
As topic says, YouTube loudness equalization isn't possible to have disabled with Iridium anylonger unfortunately and started probably as of couple days ago, it was the main reason I've installed Iridium for so would appriciate if sometimes in the future this gets an oversight if it might be able to be fixed still, would be appriciated!

**Comment by DCWH** [comment]
Same problem here. I'm using firefox, I have tried reinstalling the add-on and resetting firefox. Sadly it is still not working. Maybe YouTube has changed something.

**Comment by mkwstar** [comment]
I am having the same problem, but I found a way to kinda-bypass this for the time being.

Rightclick on the Player while the video is running, the YouTube rightclick menu (with statistics etc.) should come up ->
rightclick again to see the default dropdown menu, then click "show controls"
You then have a second overlay with it's own volume slider, which is how youtube uses it's loudness equalization.

You have to do this manually with every video so this is far from optimal but it's atleast something until this gets fixed. This is also the reason I am using this so it would be nice to see this fixed.

**Comment by RPGWiZ4RD** [comment]
That's at least better than nothing, thanks for the headsup :)

**Comment by MrPurpleGuy1987** [comment]
That doesnt work for me, as YT's controls cover it up.

**Comment by bskyb-backport** [comment]
right clicking the video indeed makes the video louder but it is sill staying normalized - when i download video file through download manager app and play it on PC player it has much wider dynamic range especially when it is a music video you can hear the instruments and vocals much clearer. There are some scripts that replace default youtube player with default browser HTML5 UI and that helps to bypass the normalization and regain clarity without downloading each video file on your harddrive. Also on mobile version of youtube normalization seems to not work either and often youtube can use default browser HTML5 UI without the need for any scripts. Youtube TV version seems to use normalization even on mobile devices. In short i wish  youtube  only forced normalization for ads and let users to manually specify if they want to normalize regular videos as well

**Comment by Onurtag** [comment]
For now I am using a bandaid fix:
https://gist.github.com/Onurtag/28d8826e2f7e369a3fe57bd69bb61f8f  
The bandaid part is that it sometimes takes a second to apply when the tab is on the background as the tab gets throttled.

Subscribe to this issue so if any of us finds a working fix or a new addon to fix it we can all know.

**Comment by bskyb-backport** [comment]
i found similar script that creates volume editor box below the video where you can specify the HTML5 volume manually to any level from 0 to 100 but as i said previously this only make the video louder but quality is still bad so if you want to increase the quality of the audio you need to completely bypass the default youtube player with  the solutions  i said before

**Comment by St0RM53** [comment]
@Onurtag Thanks man. I noticed this problems months ago and thought it was a windows issue since if you check the volume output in mixer firefox shows full but your output device don't. Today it was annoying me again and happened to noticed in the stats box a normalization value. Then narrowed down though testing and it was stupid youtube. Tried finding an extension, ended up to iridium which didn't work, checked the issues here to see if it has been reported and ended up finding your comment and after installing it works perfect. Also the enhanced-h264ify doesn't seem to be compatible with Enhancer for YouTube as vp9 was still playing.
